src/py42/services/storage/_service_factory.py

```diff
@@ -36,6 +37,11 @@ def create_preservation_data_service(self, host_address):
         streaming_connection = Connection.from_host_address(host_address)
         return StoragePreservationDataService(main_connection, streaming_connection)
 
+    def create_exfiltrated_data_service(self, host_address):
+        main_connection = self._connection.clone(host_address)
+        streaming_connection = Connection.from_host_address(host_address)
```

Comment:
I see we did this for pds as well, but what is the benefit of having separate sessions for streaming?

Reply:
Streaming only works if you use anonymous auth.